### DISCUSSION TITLE
Usage on Android

### DISCUSSION THREAD

**Discussion by nitanmarcel** [JakeWharton/mosaic#350] [Q&A]
I just found out about this and it's exactly what I need for my android application, the big question is, it is made to be usable on android compose applications or it's main target is desktop? And if it works on android, does anyone has an usage example there?

Thanks.

**Comment by JakeWharton** [comment]
You want to use it on the command line (like through `adb shell`)? Or within an actual app (like inside a `TextView`)?

**Reply by nitanmarcel** [reply]
Inside a TextView. But I'm afraid I understood the project scope wrong.

**Reply by nitanmarcel** [reply]
I actually needed something I can build an terminal UI for an app. No idea if it's possible to easily do this with the library

**Reply by JakeWharton** [reply]
No that's not what this is for. This is for writing programs that run within a terminal and whose output is solely text.
